src/Core/src/Handlers/HybridWebView/HybridWebViewHandler.Windows.cs

```diff
@@ -221,13 +221,21 @@ private async Task<bool> TryInitializeWebView2(WebView2 webView)
 			{
 				await webView.EnsureCoreWebView2Async();
 
+				var customSettings = VirtualView.InitializingWebView() as WebViewInitializingEventArgs;
+
 				webView.CoreWebView2.Settings.AreDevToolsEnabled = Handler?.DeveloperTools.Enabled ?? false;
 				webView.CoreWebView2.Settings.IsWebMessageEnabled = true;
+
+				// Allow additional custom settings to be applied through the WebViewInitializingEventArgs Class
+				customSettings?.AdditionalSettings?.Invoke(webView.CoreWebView2.Settings);
+
 				webView.CoreWebView2.AddWebResourceRequestedFilter($"{AppOrigin}*", CoreWebView2WebResourceContext.All);
 
 				webView.WebMessageReceived += OnWebMessageReceived;
 				webView.CoreWebView2.WebResourceRequested += OnWebResourceRequested;
 
+				VirtualView.InitializedWebView(webView);
+
```

Comment:
iOS, Catalyst, and Android have the Initializing and Initialized take place in the CreatePlatformView override, but on Windows, it looks like the settings take place here in the indirect call of the ConnectHandler